metricbeat/helper/prometheus/metric.go

```diff
@@ -195,6 +195,13 @@ func (m *commonMetric) GetValue(metric *dto.Metric) interface{} {
 		return value
 	}
 
+	untyped := metric.GetUntyped()
+	if untyped != nil {
+		if !math.IsNaN(untyped.GetValue()) && !math.IsInf(untyped.GetValue(), 0) {
+			return untyped.GetValue()
+		}
+	}
```

Comment:
This little code adds support to untyped in the prometheus helper, isn't it? :slightly_smiling_face:  It may worth to mention it to the developer changelog. And this would need to additional tests.

Reply:
@jsoriano hmm yeah good point, now Im thinking should this even be here. Is there a reason why untyped is not supported at the beginning?

Reply:
No idea, perhaps this hasn't been needed in the helper yet. You can leave it for a separate PR if you prefer.

Reply:
Yeah I think separate PR is probably more appropriate in this case. Thanks!